config/settings.yml

```diff
@@ -13,6 +13,7 @@ govuk_notify:
   magic_link_email_template_id: please_change_me
   course_withdraw_email_template_id: please_change_me
   course_vacancies_filled_email_template_id: please_change_me
+  course_subjects_updated_email_template_id: please_change_me
```

Comment:
The comment on `config/settings/test.yml` should be here, can we put the real templated IDs in here pls.